openhtf/output/callbacks/__init__.py

```diff
@@ -21,6 +21,12 @@
 """
 
 import collections
+
+try:
+  collectionsAbc = collections.abc
```

Comment:
Please use the six import for now.

from six.moves import collections_abc

...
collections_abc.Iterable

Reply:
Made update, PTAL

Reply:
Updated import order